docs/application/dotnet/guides/nui/flexcontainer.md

```diff
@@ -145,16 +154,18 @@ The following example shows how to create a gallery-like flexbox layout using th
     nextButton.Label = labelMap2;
     ```
 
-8.  Add the image to the content area:
+9.  Add the image to the content area:
 
-    ```
+    ```csharp
     /// Add an image to the center of the content area
-    ImageView imageView = new ImageView("image.jpg");
+    ImageView imageView = new ImageView(DirectoryInfo.Resource + "image.png");
     imageView.ParentOrigin = ParentOrigin.TopLeft;
     imageView.PivotPoint = PivotPoint.TopLeft;
     content.Add(imageView);
     ```
 
+Full example can be found here: ...
+
```

Comment:
I'm waiting for merging of this example https://github.com/Samsung/Tizen-CSharp-Samples/pull/176, after that I will add missing link here. This is just a copy of above snippets. It will be more conviniece for user to download it from here and run.

Reply:
@mijin85cho yes, thats the form we use to let readers know that there is more to it. I actually agree with your opinion. 
As @mijin85cho suggested, we can use it in the following form:
```suggestion 
For more information, see [the full example](download link).